star/test_suite/1.3M_ms_high_Z/README.rst

```diff
@@ -17,120 +19,20 @@ This test case has two parts.
 
 The final model, click on the image for a larger version, shows
 
-.. image:: ../../../star/test_suite/1.3M_ms_high_Z/docs/grid1_221.svg
-   :scale: 100%
```

Comment:
Note: `scale` doesn't actually work on the svg files, because these optimized svg files do not have a size property. Use `width` instead